frontend/src/utils/firearms-utils/arme-alarme.ts

```diff
@@ -30,84 +30,97 @@ export const arme_alarme = {
   displayLabel: 'Arme d\'alarme',
   category: 'D',
   mention: 'Libre d\'acquisition et de détention',
-  options: {
-    arme_1: {
+  options: [
+    {
       label: '1. ZORAKI R2',
       value: 'arme_1',
```

Comment:
use name of firearm rather than number in value to avoid getting mixed up in case you change their order